### PR TITLE
docs: update README.md to clarify install instructions

### PR DESCRIPTION
**Pull request by cloudwheels**
Docs only updated to address issues #10 and #32 (close on merge)

### CODE REVIEW

**Comment by cloudwheels** [comment]
Actually I wasn't aware of the work on #30 so the updates related to MN registration may not be appropriate.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # MN Bootstrap
 
-> Distribution package for Dash Masternode installation
+> Distribution package for Dash masternode installation
+
+> Note: This Docker Compose project creates a full node. Additional steps are required to register a masternode, as described in the [setup instructions](https://docs.dash.org/en/stable/masternodes/setup.html#register-your-masternode). It is not currently necessary to complete these registration steps to test development with Dash Platform services or participate as a full node on Evonet.
```
Comment (shumkov) [inline README.md:5]: I would reword this part "test development" -> "local development" since we have "local" preset

Comment (cloudwheels) [reply README.md]: @shumkov see https://github.com/dashevo/mn-bootstrap/pull/33/commits/89909c61ccdb995f467da932bded2cd3681d41f6

(and  minor change https://github.com/dashevo/mn-bootstrap/pull/33/commits/b61b11078e43573a195e1ec33c0f6a732c557a62)
